infra/src/main/kotlin/fi/fta/geoviite/infra/error/ClientException.kt

```diff
@@ -7,24 +7,26 @@ import org.springframework.http.HttpStatus
 import org.springframework.http.HttpStatus.*
 import kotlin.reflect.KClass
 
+typealias LocalizationParams = Map<String, String?>
```

Comment:
Mielipideasioita, mutta minusta perus kollektiotyyppien typealias ei paranna luettavuutta vaan heikentää sitä. Map on selkeä avain-arvo-rakenne ja käyttöpaikoissa kuitenkin sanotaan vain `mapOf(...)` eikä `LocalizationParams(...)` joten tuo typealias minusta vain hämmentää.

Sen sijaan jos tuo olisi ihan oikea oma tyyppinsä joka esim validoi/sanitoi sisältöä init-blokissa tai estää jotenkin laittamasta tuonne jotain epäkelpoa, niin sillä olisi selvemmin arvoa.

Reply:
Niin, että LocalizationParams pitäis sisällään sitten mapin, esim data class LocalizationParams(val params: Map...)?